pages/DSNP/Types/GraphChange.md

```diff
@@ -31,6 +31,8 @@ Different change types have different meanings
 | ----- |----- | ----------- |
 | 0 | Unfollow | Remove a Follow relationship |
 | 1 | Follow | Create a Follow relationship |
+| 2 | Block | Create a Block relationship |
```

Comment:
We haven't yet decided to add Block/Unblock to the protocol. I think we are _mostly_ against it being in the protocol but it's still not decided.